packages/react-reconciler/src/__tests__/ReactIncrementalUpdates-test.js

```diff
@@ -156,23 +156,11 @@ describe('ReactIncrementalUpdates', () => {
     }
 
     // Schedule some async updates
-    if (
```

Comment:
Any of these tests that change to use just `startTransition` need reverted and we need to keep the `forceConcurrentByDefaultForTesting` flag. This flag is the only thing that tests the `unstable_concurrentUpdatesByDefault` option for `createRoot` that www uses, and these tests verify that the concurrent by default behavior doesn't regress.

Reply:
I think in reality the behavior has already regressed with enableUnifiedSyncLane set to true. Do you think we can get rid of unstable_concurrentUpdatesByDefault?

Reply:
How has it regressed?

We were waiting for `enableUnifiedSyncLane` to re-test `unstable_concurrentUpdatesByDefault` so we can remove it.

Reply:
I think it is behaving the same as this test case, that even `unstable_concurrentUpdatesByDefault` is `true`, with `enableUnifiedSyncLane` enabled for all, updates all get batched and flushed together

Reply:
I see, so instead of making it always use `startTransition` can you update it to _not_ use `startTransition`? Then the `concurrentUpdatesByDefault` would still test the behavior without `startTransition`.

But even if _this_ test could be fixed, we have tests that confirm `root.render` yields when no using `startTransition` and we need to keep at least those tests.